proxysql-admin

```diff
@@ -1021,6 +1023,45 @@ function get_all_hostgroups_from_writer_hostgroup()
   echo "$hg_list"
 }
 
+# Update mysql server version details in proxysql db 
+# Globals:
+#   WRITER_HOSTGROUP_ID
+#   CLUSTER_HOSTNAME (overwrites)
+#   CLUSTER_PORT (overwrites)
+#
+# Arguments:
+#   None
+
```

Comment:
Minor nitpick, should follow the formatting for the other functions.  No blank lines before the function name.

Reply:
Done